dotnet/src/SemanticKernel.Abstractions/AI/FunctionChoiceBehaviors/NoneFunctionChoiceBehavior.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// Represents <see cref="FunctionChoiceBehavior"/> that does not provides any <see cref="Kernel"/>'s plugins' function information to the model by default.
+/// This behavior forces the model to not call any functions and only generate a user-facing message.
+/// </summary>
+/// <remarks>
+/// Although this behavior prevents the model from calling any functions, the model can use the provided function information
+/// to describe how it would complete the prompt if it had the ability to call the functions.
+/// </remarks>
+[Experimental("SKEXP0001")]
+public sealed class NoneFunctionChoiceBehavior : FunctionChoiceBehavior
```

Comment:
Can this be internal?

Reply:
Yep, it can as explained in the https://github.com/microsoft/semantic-kernel/pull/6083#discussion_r1622840916 comment.

Reply:
done